### ISSUE TITLE
pip install fails - missing http_parser.h

### ISSUE THREAD

**Issue by carlodri** [jonashaag/bjoern#224]
pip-installing v3.2.0 fails with this output:

```
      running build
      running build_py
      running build_ext
      building '_bjoern' extension
      In file included from bjoern/_bjoernmodule.c:3:
      In file included from bjoern/wsgi.h:2:
      bjoern/request.h:5:10: fatal error: 'http_parser.h' file not found
      #include "http_parser.h"
               ^~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
```

**Comment by MichaelTurekCGI** [comment]
Getting same as OP.

**Comment by jonashaag** [comment]
Thanks for reporting, this should be fixed with release 3.2.1
